feat(guide): provide 1.3 to 8.0 update guide (#803)
camunda-cloud/camunda-cloud-documentation: docs/guides/update-guide/130-to-800.md

---
id: 130-to-800
title: Update 1.3 to 8.0

> description: "Review which adjustments must be made to migrate from Camunda Cloud 1.3.x to Camunda Platform 8.0.y"

Reviewer: ```suggestion
description: "Review which adjustments must be made to migrate from Camunda Cloud 1.3.x to Camunda Platform 8.0.0"
```

#### Experimental: Consistency Checks

To configure if the basic operations on RocksDB, such as inserting or deleting key-value pairs, should check preconditions, for example that a key does not already exist when inserting.

- application.yaml
  - Broker: `zeebe.broker.experimental.consistencyChecks.enablePreconditions`
- Environment variable:
  - Broker: `ZEEBE_BROKER_EXPERIMENTAL_CONSISTENCYCHECKS_ENABLEPRECONDITIONS`

To configure if inserting or updating key-value pairs on RocksDB should check that foreign keys exist.

- application.yaml
  - Broker: `zeebe.broker.experimental.consistencyChecks.enableForeignKeyChecks`
- Environment variable:
  - Broker: `ZEEBE_BROKER_EXPERIMENTAL_CONSISTENCYCHECKS_ENABLEFOREIGNKEYCHECKS`

### Operate

With Camunda Platform 8 the IAM component was replaced with the new [Identity](/self-managed/identity/what-is-identity.md) project. There is no update path from IAM to Identity, which means all configured roles and permissions have to be recreated after the update. Please refer to the [Operate documentation](/self-managed/operate-deployment/authentication.md#identity) how to connect Operate with Identity.

> Operate supports with version 8.0 to configure a secure connections to the Zeebe gateway, see the [connection configuration section](/self-managed/operate-deployment/configuration.md#settings-to-connect-1) in the Operate documentation.

Reviewer: ```suggestion
With version 8.0. Operate supports configuring secure connections to the Zeebe gateway, see the [connection configuration section](/self-managed/operate-deployment/configuration.md#settings-to-connect-1) in the Operate documentation.
```

> Operate offers with version 8.0 a new [REST API](/apis-clients/operate-api/index.md) to access process data.

Reviewer: ```suggestion
With version 8.0, Operate offers a new [REST API](/apis-clients/operate-api/index.md) to access process data.
```

### Tasklist

With Camunda Platform 8 the IAM component was replaced with the new [Identity](/self-managed/identity/what-is-identity.md) project. There is no update path from IAM to Identity, which means all configured roles and permissions have to be recreated after the update. Please refer to the [Tasklist documentation](/self-managed/tasklist-deployment/authentication.md#identity) how to connect Tasklist with Identity.

> Tasklist supports with version 8.0 to configure a secure connections to the Zeebe gateway, see the [connection configuration section](/self-managed/tasklist-deployment/configuration.md#settings-to-connect-1) in the Tasklist documentation.

Reviewer: ```suggestion
With version 8.0, Tasklist supports configuring secure connections to the Zeebe gateway, see the [connection configuration section](/self-managed/tasklist-deployment/configuration.md#settings-to-connect-1) in the Tasklist documentation.
```

### IAM

With Camunda Platform 8 the IAM component was replaced with the new [Identity](/self-managed/identity/what-is-identity.md) project. Please refer to the [Identity documentation](/self-managed/identity/getting-started/index.md) to learn how to install and configure it.

## Client

### Zeebe

> Zeebe 8.0 adds the possibility to evaluated decisions, therefore [new deployment command](/apis-clients/grpc.md#deployresource-rpc) was introduced to better support DMN files during deployment. The [old deployment command](/apis-clients/grpc.md#deployprocess-rpc) is deprecated and we recommend to migrate your application code to use the new deployment function.

Reviewer: ```suggestion
Zeebe 8.0 adds the possibility to evaluate decisions, therefore the [new deployment command](/apis-clients/grpc.md#deployresource-rpc) was introduced to better support DMN files during deployment. The [old deployment command](/apis-clients/grpc.md#deployprocess-rpc) is deprecated and we recommend to migrate your application code to use the new deployment function.
```
